msal/tests/dev apps/XForms/XForms/AcquirePage.xaml

```diff
@@ -65,14 +65,17 @@
             </StackLayout>
         </Frame>
 
-        <Label Text="Result:" Font="Bold,15"/>
+        <Label x:Name="acquireResponseTitleLabel" Text="Result:" Font="Bold,15"  AutomationId="testResult"/>
         
         <ScrollView Orientation="Vertical" VerticalOptions="FillAndExpand">
             <StackLayout Orientation="Vertical" VerticalOptions="FillAndExpand">
                 <Frame OutlineColor="Black" Padding="10">
-                    <Label x:Name="acquireResponseLabel" Text="" />
+                    <Label x:Name="acquireResponseLabel" Text=""/>
                 </Frame>
-                <Button  Text="Clear"  Clicked="OnClearClicked" TextColor="Blue" Font="Bold,15"/>
+                <StackLayout Orientation="Horizontal"  HorizontalOptions="FillAndExpand">
+                    <Button  Text="Clear Result"  Clicked="OnClearClicked" TextColor="Blue" Font="Bold,15"/>
+                    <Button  Text="Clear Cache"  Clicked="OnClearCacheClicked" TextColor="Blue" Font="Bold,15" AutomationId="clearCacheButton"/>
```

Comment:
any reason you are not setting AutomationId on all elements?

Reply:
The UI framework does not seem to work well with the tabbed page view that is used for the MSAL app. it can only access the front page thus, I will add IDs for all elements on the front page. We will most likely need to build an application that is dedicated to UI testing and is the same across both libraries.